### ISSUE TITLE
how to scale rendered email html to fit screen of mobile device

### ISSUE THREAD

**Issue by abbood** [MailCore/mailcore2#351]
running the code from the [ios test](https://github.com/MailCore/mailcore2/tree/master/example/ios/iOS%20UI%20Test/iOS%20UI%20Test) included in this repo.. i noticed emails rendered in html did not scale to fit the screen (it appeared outside the bound of the screen)..

I tried setting the webview displaying the html to have a `scalesPageToFit = YES`. Now that worked, but it exposed the following problem: Suppose the content of the email has a fixed `<div>` width of 500px.. and it's parent has a value of 100%. the Iphone screen width is 640px wide.. so the result will be that the screen fits the 100% of the parent div.. but the remaining 640-500 = 140px will be left empty.. 

You can see the problem here (compare how the same email is rendered between the mailcore iOS test [sample](https://plus.google.com/u/0/photos/112715343588929261063/albums/5919710846280470001/5919710854587530482?pid=5919710854587530482&oid=112715343588929261063) and how it's [rendered](https://plus.google.com/u/0/photos/112715343588929261063/albums/5919710846280470001/5919710847656038194?pid=5919710847656038194&oid=112715343588929261063) in the iOS gmail app. 

long story short: Is there an existing library or an existing solution that would optimally scale the width of the `div` that holds the interesting content? Or will I have to just figure it out on my own? This doesn't seem like a trivial matter to me.
